.github/workflows/build.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       - name: Install VSCE
         shell: bash
         run: |
+          npm install -g patch-package
```

Comment:
is this needed?
patch-package is used for the testing library happy-dom in chat, but that's only a dev-dep?